File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/EntitiesBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author yudong
 * @date 2023/2/12
 */
@NoArgsConstructor
@Data
public class EntitiesBean {
    private String title;
    private List<FieldsBean> fields;
    private List<IndexsBean> indexs;
    private List<?> headers;
    private String chnname;
    private String originalCreateTableSql;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/EntitiesBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */


@AllOpen
class EntitiesBean {
    var title: String? = null
    var fields: List<FieldsBean>? = null
    var indexs: List<IndexsBean>? = null
    var headers: List<*>? = null
    var chnname: String? = null
    var originalCreateTableSql: String? = null
}
